src/storage/SimpleResourceStore.ts

```diff
@@ -128,10 +128,11 @@ export class SimpleResourceStore implements ResourceStore {
    * @returns The resulting Representation.
    */
   private generateRepresentation(data: Quad[], preferences: RepresentationPreferences): Representation {
-    if (preferences.type && preferences.type.some((preference): boolean => preference.value.includes('text/turtle'))) {
-      return this.generateBinaryRepresentation(data);
+    // Always return turtle unless explicitly asked for quads
```

Comment:
The main idea was that this is not an actual `ResourceStore` but just a dummy until we get the real ones (that's why the `Simple` indicators are there) and this makes most things work. This is now a case of the store returning turtle if the accept header is `*/*` instead of the internal quads format (for which we have a type now as well).

Reply:
OK; maybe just have a comment about this (recall that others might start making stores).